source/common/grpc/async_client_manager_impl.cc

```diff
@@ -15,6 +15,8 @@
 #include "source/common/grpc/google_async_client_impl.h"
 #endif
 
+static constexpr uint64_t DefaultEntryIdleDuration{50000};
```

Comment:
nit: please move into the unnamed namespace if possible.

Reply:
done